src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java

```diff
@@ -279,34 +277,8 @@ private static FilePath tempDir(FilePath ws) {
             this.toolName = toolName;
         }
 
-        private void stopContainer(StepContext context) {
-            Launcher launcher;
-            TaskListener listener;
-            try {
-                launcher = context.get(Launcher.class);
-                listener = context.get(TaskListener.class);
-            } catch (Exception x2) {
-                LOGGER.log(Level.WARNING, null, x2);
-                return;
-            }
-            try {
-                EnvVars launcherEnv = context.get(EnvVars.class);
-                Node node = context.get(Node.class);
-                destroy(container, launcher, node, launcherEnv, toolName);
-            } catch (Exception x) {
-                x.printStackTrace(listener.error("Could not kill container " + container));
-            }
-        }
-
-        @Override public void onSuccess(StepContext context, Object result) {
-            stopContainer(context);
-            context.onSuccess(result);
-        }
-
-        @Override public void onFailure(StepContext context, Throwable t) {
-            LOGGER.log(Level.FINE, "execution failure container=" + container, t);
```

Comment:
Was added without explanation in #34. Should already have been logged elsewhere anyway.